dashboard/config/locales/scripts.en.yml

```diff
@@ -17525,7 +17525,7 @@ en:
             Project - Digital Information Dilemmas Part 2:
               name: Project - Digital Information Dilemmas Part 2
             'Lesson 14: Assessment Day':
-              name: 'Lesson 14: Assessment Day'
+              name: Assessment Day
```

Comment:
this update looks like it is part of csd2-projects-temp, not csd2-2020.

Reply:
I see what you are saying. I think something weird is up with this diff. These changes were generated by changing the script files and then reseeding. Those lines are not all this close together in the real file... not sure whats going on. I can look into more.

Reply:
Ah, sorry, I eventually was able to review the changes using a different tool, and forgot to remove this comment. Changes to this file look good!